bin/pycbc_multi_inspiral

```diff
@@ -615,6 +640,12 @@ with ctx:
                 snr_dict[ifo] = (
                     snr_ts[matched_filter[ifo].segments[s_num].analyze] * norm
                 )
+                # Raise error if this segment has no data
+                if len(snr_dict[ifo]) == 0:
+                    raise RuntimeError(
+                        'The SNR triggers dictionary is empty. This '
+                        'should not be possible.'
+                    )
```

Comment:
We discussed this a bit on Slack, and I am still not quite understanding this check. `snr_dict[ifo]` seems to be a (portion of) SNR time series. So first of all, the error message is not correct, as we are not looking at triggers here. Second, since the error condition should never occur in *any* kind of normal operation, this check is better expressed using `assert` rather than an if statement and `raise`. So I suggest
```suggestion
                assert len(snr_dict[ifo]) > 0, f'SNR time series for {ifo} is empty'
```